{"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState\/EndReason\/failed(withError:)"},"schemaVersion":{"minor":3,"major":0,"patch":0},"metadata":{"modules":[{"name":"KaleyraVideoSDK"}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failed"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"withError"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"symbolKind":"case","roleHeading":"Case","role":"symbol","externalID":"s:15KaleyraVideoSDK9CallStateO9EndReasonO6failedyAEs5Error_p_tcAEmF","title":"CallState.EndReason.failed(withError:)"},"hierarchy":{"paths":[["doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK","doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState","doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState\/EndReason"]]},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"failed","kind":"identifier"},{"kind":"text","text":"("},{"kind":"externalParam","text":"withError"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"type":"text","text":"Call has ended because of an error."}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/kaleyravideosdk\/callstate\/endreason\/failed(witherror:)"]}],"references":{"doc://com.kaleyra.KaleyraVideoSDK/documentation/KaleyraVideoSDK/CallState":{"navigatorTitle":[{"text":"CallState","kind":"identifier"}],"title":"CallState","url":"\/documentation\/kaleyravideosdk\/callstate","role":"symbol","kind":"symbol","type":"topic","abstract":[{"text":"An enumeration representing the states a call can be in.","type":"text"}],"identifier":"doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"CallState"}]},"doc://com.kaleyra.KaleyraVideoSDK/documentation/KaleyraVideoSDK/CallState/EndReason":{"identifier":"doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState\/EndReason","abstract":[{"type":"text","text":"An enumeration representing the possible call end reasons."}],"url":"\/documentation\/kaleyravideosdk\/callstate\/endreason","kind":"symbol","title":"CallState.EndReason","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"EndReason"}],"type":"topic","role":"symbol","navigatorTitle":[{"text":"EndReason","kind":"identifier"}]},"doc://com.kaleyra.KaleyraVideoSDK/documentation/KaleyraVideoSDK/CallState/EndReason/failed(withError:)":{"abstract":[{"text":"Call has ended because of an error.","type":"text"}],"identifier":"doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK\/CallState\/EndReason\/failed(withError:)","url":"\/documentation\/kaleyravideosdk\/callstate\/endreason\/failed(witherror:)","kind":"symbol","title":"CallState.EndReason.failed(withError:)","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"failed","kind":"identifier"},{"kind":"text","text":"("},{"text":"withError","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"text":")","kind":"text"}],"type":"topic","role":"symbol"},"doc://com.kaleyra.KaleyraVideoSDK/documentation/KaleyraVideoSDK":{"title":"KaleyraVideoSDK","url":"\/documentation\/kaleyravideosdk","role":"collection","kind":"symbol","type":"topic","abstract":[],"identifier":"doc:\/\/com.kaleyra.KaleyraVideoSDK\/documentation\/KaleyraVideoSDK"}}}